_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -1,43 +1,38 @@
+# Something New
 
+There are only 8 months, 2 days, and 4 hours until check-in, but who's counting? HackIllinois 2019 will be the sixth time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. Our mission this year:
 
-# Something New:
+>**To create an environment conducive to education, collaboration, and mentorship within the Open Source Community**
 
-There are only 8 months, 2 days, and 4 hours until check-in, but who’s counting? HackIllinois 2019 will be the 6th time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. We have established our mission this year:
+We _create an enviroment_  in that our event is not defined by a simple "code for a prize" model. We plan to pack the entire event with content for attendees, from Illinois and over a dozen other universities, so that they can have a unique experience throughout the weekend. HackIllinois carries development, social, and networking opportunities, and these establish the three pillars -- _education, collaboration, and mentorship_ -- of our event. The theme this year follows the trail that has been blazed by our past directors and staff, but we wanted to assert our post _within the Open Source community_. We fully believe that our event betters the Open Source Community: by bringing together new students, continuing development on projects we believe in, and showcasing the talent pool that we have to offer.
 
-**_To create an environment conducive to education, collaboration, and mentorship within the Open Source Community_**
+More specifically, thie year we want to make HackIllinois even more _exciting_. We are putting as much emphasis as we can on making this event more than 36 hours of coding; simply learning about open source is not enough at HackIllinois 2019. It will be the immersive experience of teamwork, guidance, and excitement that hackers have come to expect out of our staff. We will run interactive workshops, present lively mini-events, and host top industry talent for HackIllinois 2019. The Open Source experience will be a cut above that of the past: exciting to both the OSS gurus and those who have not yet explored this region of the technology world.
```

Comment:
exciting isn't necessarily a specific word - maybe replace this with engaging or something else (maybe reword sentence and take our more specifically)
"thie" --> "this"